packages/smooth_app/lib/pages/user_management/forgot_password_page.dart

```diff
@@ -208,7 +191,7 @@ class _ForgotPasswordPageState extends State<ForgotPasswordPage>
                           : appLocalizations.send_reset_password_mail,
                       style: theme.textTheme.bodyText2?.copyWith(
                         fontSize: 18.0,
-                        color: theme.colorScheme.surface,
+                        color: theme.colorScheme.onPrimary,
```

Comment:
Again I'm surprised: no specified color for the background should mean no specified color for the foreground.